### DISCUSSION TITLE
How to prevent the "as completed" in email notifications when closing issues which are not completed (but rejected, for example)?

### DISCUSSION THREAD

**Discussion by sypets** [community/community#38402] [Projects and Issues] [answered]
This is about email notifications concerning closed issues.

When issues are closed, GitHub always sends a "Closed ... as completed" email (in the repositories I am watching). What is bothering me is not the email itself, but the "**as completed**" - whether it actually was completed or not.

It is nice to have a concise message in the email, so you don't have to visit the issue, but ideally, it should be correct.

This is also sent this way when issues are manually closed. Why does GitHub assume they are completed here, and is there a way to prevent the "as completed"?

IMHO, it should only be added if there is a commit merged which causes the issue to be closed (e.g. via the keyword "Resolves", etc.).

Alternatively, it might be nice to prevent this by adding labels such as "Rejected" etc. to the issue, however AFAIK labels do not have special meanings and can be used randomly (not 100% sure here).

----

I am new, have done a search and read the guidelines.

**Comment by airtower-luna** [comment, marked as answer]
When you close the issue manually you can choose the reason (click on that little triangle pointing down):
![grafik](https://user-images.githubusercontent.com/5814919/200163002-f35c29ed-424a-4698-b42d-fb25750dffab.png)

If you [close via API](https://docs.github.com/en/rest/issues/issues#update-an-issue) you can use the `state_reason` parameter for the same purpose.

I hope that helps! :slightly_smiling_face:

**Reply by AllanGibbs** [reply]
And how can I simply comment on an issue? I opened an issue. Somebody commented on it. I wanted to reply. In the text field I can leave a comment but I have to choose "Close issue" or "Close as not planned" and then click on the green button "Comment". But I do not want to close the issue, I just want to comment.
Is it only possible by replying to the e-mail? Somehow I managed just to comment without closing. I am confused.

https://github.com/overload-development-community/olmod/issues/270

**Reply by airtower-luna** [reply]
Use the same text field, just press the "comment" button instead of the "close" one. :wink:

**Reply by AllanGibbs** [reply]
OK. Thank you, Airtower. I'll try it next time.
